helm-charts/templates/control-plane/postgres/initdb-conf.yaml

```diff
@@ -85,6 +85,7 @@ data:
         LOG_LEVEL VARCHAR(255) DEFAULT 'OFF',
         REVISIONS_CREATED INTEGER DEFAULT 0,
         SDK JSONB,
+        CATEGORIES JSONB,
```

Comment:
shall we update database/postgres/db.sql file as well

Reply:
fix 6b17f1283ecb30a54972e46726f075c5500fdff4